test/extensions/filters/network/mysql_proxy/mysql_integration_test.cc

```diff
@@ -1,5 +1,8 @@
+#ifndef WIN32
 #include <pthread.h>
```

Comment:
Is this actually needed on linux? I kind of doubt it. Can it just be removed?

Reply:
Confirmed on gcc.